src/pickup.cpp

```diff
@@ -214,7 +215,32 @@ bool pick_one_up( item_location &loc, int quantity, bool &got_water, bool &offer
         picked_up = true;
         option = NUM_ANSWERS; //Skip the options part
     } else if( newit.made_of_from_type( LIQUID ) ) {
-        got_water = true;
+
+        //If the item we're trying to pick up is a frozen liquid check if we have a hammering tool and if the user wants to harvest the liquid
```

Comment:
Yes, that's what the y code does, why repeat the code as comment?

Reply:
Removed redundant comment